integration-tests/validation/src/main/res/values/gen_strings.xml

```diff
@@ -1,15 +1,42 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- DO NOT EDIT. AUTO GENERATED BY DESIGN COMPOSE PLUGIN. -->
 <resources>
-  <string name="label_hello" translatable="true">Hello,</string>
-  <string name="label_goodbye" translatable="true">Goodbye,</string>
-  <string name="label_goodbye_1" translatable="true">Goodbye</string>
-  <string name="label_parked" translatable="true">Parked</string>
-  <string name="label_reverse" translatable="true">Reverse</string>
-  <string name="label_charging" translatable="true">Charging</string>
-  <string name="label_driving" translatable="true">Driving</string>
-  <string name="label_regen" translatable="true">Regen</string>
-  <string-array name="label_styled_text" translatable="true">
+  <string name="label_hello"
+    description="Hello with a comma"
+    charlimit="20"
```

Comment:
Are `description` and `charlimit` official keys, or are they just made up but Android is ok with it?

Reply:
Kind of made up but Android is ok with it. It compiles without error. And there are some other apps using description the same way: https://source.corp.google.com/piper///depot/google3/java/com/google/android/gmscore/integ/modules/tapandpay/res/res/values/strings.xml;l=17?q=strings.xml%20description

Using `description` and `charlimit` will be easier for me to parse the input xml file.